### ISSUE TITLE
version not loading

### ISSUE THREAD

**Issue by PakGamesShow** [Ventile-Client/Download#4]
hi i have an issue that my bedrock mc is in 1.17.34 version i want to play with surge dll but when i go and download version 1.17.11 on ventile client and click slelect it dosent change the version can still is on 1.17.34 pls tell me the solution i really want to play on surge

**Comment by DeathlyBower959** [comment]
Ik this is a late comment, but if it is still working, could you let me know? I dont check GitHub issues often. If its not, then this next update (4.2.1) should fix it i believe

**Comment by PakGamesShow** [comment]
yup i updated it and it succssesfully loaded but when i selected 1.17.1 and
launched mc but when it is loaded the massahe appear sign up i clicked
close and it just corrupted mc closed it self and when i tried to open mc
again it just open and close then i deleted mc and download again from
microsoft tell me how to fix it

**Comment by DeathlyBower959** [comment]
Did you wait for the select button to become ungreyed?

**Comment by PakGamesShow** [comment]
Yes, I did.

**Comment by DeathlyBower959** [comment]
What was the massive sign that popped up, and you closed it?

**Comment by PakGamesShow** [comment]
i didnt close it is closed itself and how it closed it just loaded and
closed itself

**Comment by DeathlyBower959** [comment]
do you know what it said?

**Comment by PakGamesShow** [comment]
it doesnt say anything it just closes withoit saing any massage or error

**Comment by DeathlyBower959** [comment]
If you are able to. Can you screen record trying to switch versions and share it here? (streamable maybe?)

**Comment by PakGamesShow** [comment]
which is the best free screen recorder so that i can record

**Comment by DeathlyBower959** [comment]
Obs studio

**Comment by PakGamesShow** [comment]
how can i upload it here?

**Comment by PakGamesShow** [comment]
and also i have to download minecraft bedrock again ;(
